src/PaymentBundle/DependencyInjection/Configuration.php

```diff
@@ -35,6 +35,7 @@ public function getConfigTreeBuilder()
                 ->scalarNode('selector')->defaultValue('sonata.payment.selector.simple')->cannotBeEmpty()->end()
                 ->scalarNode('generator')->defaultValue('sonata.payment.generator.mysql')->cannotBeEmpty()->end()
                 ->arrayNode('transformers')
+                    ->addDefaultsIfNotSet()
```

Comment:
Ah ok, glad you pointed it out! I would like to have it in a separate PR **first**, so could you fix this bug and add the config test there? I will merge it asap and the you could rebase this PR on top of the **first** one?